src/components/SpaceProposalVote.vue

```diff
@@ -34,6 +34,23 @@ const validatedUserChoice = computed(() => {
   return null;
 });
 
+const buttonTooltip = computed(() => {
+  if (
+    props.proposal.type === 'ranked-choice' &&
+    selectedChoices.value < props.proposal.choices.length
+  )
+    return 'Please rank all choices';
+
+  if (
+    props.proposal.type !== 'approval' &&
+    props.proposal.type !== 'ranked-choice' &&
+    selectedChoices.value < 1
+  )
+    return 'Please select at least one choice';
+
+  return '';
+});
+
```

Comment:
Looks good,

one thing missing is approval voting, right now it is possible to vote without choice, but maybe it is expected?
<img width="981" alt="Untitled" src="https://github.com/snapshot-labs/snapshot/assets/15967809/44f389ec-dcbf-4220-ba5e-1c1adfa971b0">